components/ide/code/leeway.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN sudo apt-get update \
     && sudo apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GP_CODE_COMMIT ae8dc44299153ca820ac9e13cc9def5bb6baafd1
+ENV GP_CODE_COMMIT 334e0610dd1bb73487812b417b113d6a2b96e1b1
```

Comment:
**suggestion**: Maybe it's worth renaming _Open Discourse Community (Discuss and get support)_ to _Open Community Forum_. WDYT?

Reply:
yes, we can do it